resources/graphics.svg

```diff
@@ -2695,6 +2735,46 @@
          y="2612"
          id="listRemoveSmall" />
     </g>
+    <g
+       id="menus"
+       inkscape:label="menus">
+      <rect
```

Comment:
Thanks for going to the trouble of making a new section and moving existing icons into it. The long term goal is eventually to have the whole of `graphics.svg` organised like this, so it's great to see some progress.